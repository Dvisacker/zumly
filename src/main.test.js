/* eslint-env jest */

import Zumly from './main.js'

describe('Zumly class', () => {
  it('Zumly(string)', () => {
    expect(new Zumly('cool')).toMatchSnapshot()
  })
})
