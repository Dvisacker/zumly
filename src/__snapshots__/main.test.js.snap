// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`awesome class Awesome(string) 1`] = `
Awesome {
  "id": undefined,
  "string": "cool",
}
`;
